examples/example_query_policy.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+"""
+Example script showing how to use the LaceworkClient class.
+"""
+
+import logging
+import os
+import random
+import string
+
+from dotenv import load_dotenv
+from laceworksdk import LaceworkClient
+
+logging.basicConfig(level=logging.DEBUG)
+
+load_dotenv()
+
+RANDOM_TEXT = "".join(random.choice(string.ascii_uppercase) for _ in range(4))
```

Comment:
nit. I think you can also use `random.choices` here:

```
''.join(random.choices(string.ascii_uppercase, k=4))
```

Reply:
🔥 thanks man!